90zfsbootmenu/zfsbootmenu-lib.sh

```diff
@@ -2099,7 +2126,16 @@ emergency_shell() {
 
   echo -n "Launching emergency shell: "
   echo -e "${message}\n"
-  /bin/bash -l
+
+  # -i (interactive) mode will source /.bashrc
+  /bin/bash -i
```

Comment:
I'm surprised bash doesn't source `/.bashrc` in `-l` mode.

Closer review of the man page suggests that `-l` does not necessarily imply `-i`, so maybe `bash -l -i` would be better because the login shell will source `/etc/profile` automatically. You could then drop the manual sourcing in `/.bashrc`. I don't know if this makes a difference anywhere.

Reply:
It looks like if both `-l` and `-i` are passed in as arguments to bash, `-l` wins and `${HOME}/.bashrc` isn't sourced.